### PR TITLE
Added mastodon shortcode

### PR DESCRIPTION
**Pull request by alephnull**
This is my first foray into hugo. Please do let me know if I could do this better. The mastodon code is from https://www.kevingimbel.com/mastodon-embed-shortcode-for-hugo/

### CODE REVIEW

**Comment by zwbetz-gh** [comment]
Thanks for the PR. I'm away for the next week but will review once I'm back.

File: README.md [hunk 1]
```diff
@@ -164,6 +165,17 @@ class="some-class"
 style="some-style" >}}
 ```
 
+### `mastodon`
+
+This will embed a toot in an `iframe`.
+
+These arguments are optional: `width`, `height`
```
Comment (zwbetz-gh) [inline README.md]: Add a period.

File: README.md [hunk 2]
```diff
@@ -164,6 +165,17 @@ class="some-class"
...
+
+```
+{{% mastodon
+status="https://mastodon.social/@kevingimbel/100700713283716694"
+width="1000" height="500" %}}
+
```
Comment (zwbetz-gh) [inline README.md:179]: Close this markdown code block with 3 backticks.

**Comment by zwbetz-gh** [comment]
Tested your changes locally, looks good. Once you address my README comments, I think this is ready to merge.

**Comment by alephnull** [comment]
> Tested your changes locally, looks good. Once you address my README comments, I think this is ready to merge.

This is done.

**Pull request merged**